static/src/javascripts/projects/common/modules/commercial/fiv-banner.js

```diff
@@ -0,0 +1,132 @@
+// @flow
```

Comment:
this file is mostly copied from the membership-engagement-banner file but with much simplification as we don't need the AB testing from a spreadsheet functionality.